### ISSUE TITLE
cannot use with pc

### ISSUE THREAD

**Issue by junjin8433** [domi1294/BluetoothHidDemo#1]
when using your code, we can connect to another android phone, it works, but when connecting to a pc(we've tried mac os and windows), it does not work, I'm wondering maybe you know the reasons

**Comment by domi1294** [comment]
This is correct. I figured out why it does not work with PCs and I'm currently in the process of rewriting it, I'll update this issue after I updated the project

**Comment by jusss** [comment]
is this bug fixed? could you tell me why it does not work with PCs?

**Comment by N0b06y** [comment]
I would like to use this for a project. Does it work now?
